articles/build-a-jekyll-site/index.md

```diff
@@ -0,0 +1,354 @@
+    alt: image Node.js debugger docker
+---
+Building a website nowadays is not as hard or expensive as it was in past years. These days, you don't have to have to learn a crazy programming language to have your site up and running. Instead, you can go the Jamstack route. In this article we will look at building and deploying a Jekyll site.
+<!--more-->
+### Introduction
+[Jekyll](https://jekyllrb.com/) is a [Jamstack](https://jamstack.org/) static website generator. Jamstack is a **concept** of building websites that don't depend on web servers and *databases*. Jekyll was released back in 2008 and has been one of the most convenient ways of building [all types of sites](https://jekyllrb.com/showcase/). It's written in [Ruby](https://www.ruby-lang.org/en/) and uses the Liquid template engine to render pages.
+
+### Prerequisites
+To follow through this tutorial, you will need the following:
+
+1. A GitHub [account](https://github.com/). If you don't have one you can create the basics of Sne [here](https://sass-lang.com/guide).
+2. A basic knowledge of HTML and CSS.
+3. A code editor. You can use your preferred code editor or download VS Code [here](https://code.visualstudio.com/download).
+
+By the end of this tutorial, you will be able to build a Jekyll blog from scratch.
+
+### Step 1 -- Install Ruby
+Since Jekyll is written in Ruby, you will need a Ruby installation on your machine to run Jekyll.
+
+In Linux, you can install it using [Snap](https://snapcraft.io/) or [Apt](https://wiki.debian.org/Apt) package manager.
+
+1. **Install using apt package manager**.
+First, update your apt repositories to look for the latest Ruby version.
+
+	```bash
+	sudo apt-get update
+	```
+Then install Ruby.
+
+	```bash
+	$ sudo apt-get install ruby-full
+	```
+
+2. **Install using Snap package manager**.
+
+	```bash
+	$ sudo snap install ruby --classic
+	```
+
+If you are running macOS, you can install Ruby using [Homebrew](https://brew.sh/).
+
+```bash
+$ brew install ruby
+```
+
+If you are running Windows, install Ruby using [Ruby Installer](https://rubyinstaller.org/).
+To confirm the Ruby installation, run the command `ruby -v` in a terminal. The output will be similar but may vary from the one below.
+
+```bash
+ruby 2.7.0p0 (2019-12-25 revision 647ee6f091)
+```
+
+You'll need to set up a gem installation directory to prevent permission errors.
+Add the lines below at the bottom of the `.bashrc` file located in the home folder.
+
+```bash
+# Install Ruby Gems to ~/gems
+export GEM_HOME="$HOME/gems"
+export PATH="$HOME/gems/bin:$PATH"
+```
+
+### Step 2 -- Install Jekyll
+Jekyll comes as a [Ruby Gem](https://guides.rubygems.org/what-is-a-gem/) and can be installed in any system running Ruby. Run the command below in a terminal to install Jekyll.
+
+```bash
+$ gem install jekyll bundler
+```
+
+### Step 3 -- Create a new Jekyll site
+Create a new fo the basics oflSer w[here](https://sass-lang.com/guide) you will be creating your blog.
```

Comment:
started a preliminary review on your article -but I've been running into similar sentences like this - if you could revisit the article - do a full revision - checking for mistake or grammar - and improve any readability issues 🚀 

Eg: "Create a new fo the basics oflSer w[here](https://sass-lang.com/guide) you will be creating your blog."